examples/js/loaders/sea3d/SEA3DLoader.js

```diff
@@ -2289,115 +1999,129 @@ THREE.SEA3D.prototype.getVertexAnimation = function(sea) {
-	LOAD_PROGRESS:"sea3d_progress",
-	DOWNLOAD_PROGRESS:"sea3d_download",
-	COMPLETE:"sea3d_complete",
-	OBJECT_COMPLETE:"sea3d_object",
-	ERROR:"sea3d_error"
+	LOAD_PROGRESS: "sea3d_progress",
+	DOWNLOAD_PROGRESS: "sea3d_download",
+	COMPLETE: "sea3d_complete",
+	OBJECT_COMPLETE: "sea3d_object",
+	ERROR: "sea3d_error"
 }
 
-THREE.SEA3D.prototype.onProgress = null;
+THREE.SEA3D.prototype.onProgress = undefined;
 
 THREE.SEA3D.prototype.onComplete = function( args ) {
-	args.file = this.scope; args.type = THREE.SEA3D.Event.COMPLETE; 	
-	args.file.dispatchEvent(args);
-	//console.log("SEA3D:", args.message);
+
+	args.file = this.scope; args.type = THREE.SEA3D.Event.COMPLETE;
+	args.file.dispatchEvent( args );
+
 }
 
 THREE.SEA3D.prototype.onLoadProgress = function( args ) {
+
 	args.file = this.scope; args.type = THREE.SEA3D.Event.LOAD_PROGRESS;
-	args.file.dispatchEvent(args);
-	//console.log("SEA3D:", args.progress);	
-	if (args.file.onProgress) args.file.onProgress( args );
+	args.file.dispatchEvent( args );
+	if ( args.file.onProgress ) args.file.onProgress( args );
+
 }
 
 THREE.SEA3D.prototype.onDownloadProgress = function( args ) {
+
 	args.file = this.scope; args.type = THREE.SEA3D.Event.DOWNLOAD_PROGRESS;
-	args.file.dispatchEvent(args);	
-	//console.log("SEA3D:", args.progress);
-	if (args.file.onProgress) args.file.onProgress( args );
+	args.file.dispatchEvent( args );
+	if ( args.file.onProgress ) args.file.onProgress( args );
+
 }
 
 THREE.SEA3D.prototype.onCompleteObject = function( args ) {
+
 	args.file = this.scope; args.type = THREE.SEA3D.Event.OBJECT_COMPLETE;
-	args.file.dispatchEvent(args);
-	//console.log("SEA3D:", args.object.name + "." + args.object.type);
+	args.file.dispatchEvent( args );
+
 }
 
 THREE.SEA3D.prototype.onError = function( args ) {
+
 	args.file = this.scope; args.type = THREE.SEA3D.Event.ERROR;
-	args.file.dispatchEvent(args);
-	//console.log("SEA3D:", args.message);
+	args.file.dispatchEvent( args );
+
 }
 
 //
 //	Loader
 //
 
-THREE.SEA3D.prototype.load = function( url ) {			
+THREE.SEA3D.prototype.load = function( url ) {
+
 	this.loadBytes();
-	this.file.load(url);		
+	this.file.load( url );
+
+}
+
+THREE.SEA3D.prototype.onHead = function( args ) {
+
+	if ( args.sign != 'TJS' ) {
+
+		throw "Sign '" + args.sign + "' not supported! Use SEA3D Studio to export."
+
+	}
+
 }
 
-THREE.SEA3D.prototype.loadBytes = function( data ) {			
-	this.file = new SEA3D.File( data );
+THREE.SEA3D.EXTENSIONS = [];
+
+THREE.SEA3D.prototype.loadBytes = function( data ) {
+
+	this.file = new SEA3D.File();
 	this.file.scope = this;
 	this.file.onComplete = this.onComplete;
 	this.file.onProgress = this.onLoadProgress;
 	this.file.onCompleteObject = this.onCompleteObject;
 	this.file.onDownloadProgress = this.onDownloadProgress;
 	this.file.onError = this.onError;
-	
-	//	SEA3D
-	
-	switch(this.config.parser)
+	this.file.onHead = this.onHead;
+
+	// SEA3D
+
+	this.file.typeRead[ SEA3D.Geometry.prototype.type ] =
```

Comment:
Seems like I was wrong. This kind of assignment is valid (atleast in chrome)

[jsfiddle](https://jsfiddle.net/791qjwtx/)